survey_formio/models/survey_question.py

```diff
@@ -0,0 +1,186 @@
+        - datePicker.maxMode : required but no corresponding parameter in survey.
+                               Set to default form.io value
+        - timePicker.hourStep : required but no corresponding parameter in survey.
+                                Set to default form.io value
+        - timePicker.minuteStep : required but no corresponding parameter in survey.
+                                  Set to default form.io value
+        """
+        self.ensure_one()
+        return {
+            "datepicker": {"minMode": "day", "maxMode": "year"},
+            "timePicker": {"hourStep": 1, "minuteStep": 1},
+            "enableTime": is_datetime,
+        }
+
+    def _get_formio_radio_parameters(self):
+        """
+        Returns a dict containing parameters specific to a radio component
+
+        This function does not take display_mode, column_nb and comments_allowed
+        into account yet.
+
+        Form.io parameters:
+        - values: list of dict {"label": label, "value": value}
+        """
+        self.ensure_one()
+        return {
+            "values": [
+                {"label": label.value, "value": f"a{label.id}"}
+                for label in self.labels_ids
+            ]
+        }
+
+    def _get_formio_selectboxes_parameters(self):
+        """
+        Returns a dict containing parameters specific to a select boxes component
+
+        This function does not take column_nb and comments_allowed into account yet.
+
+        Form.io parameters:
+        - values: list of dict {"label": label, "value": value}
+        """
+        self.ensure_one()
+        return {
+            "values": [
+                {"label": label.value, "value": f"a{label.id}"}
+                for label in self.labels_ids
+            ]
+        }
+
+    def _get_formio_component(self):
+        """
+        Returns a dict representing a question as a valid formio component
+        """
+        self.ensure_one()
+        if self.question_type == "matrix":
+            return False
+
+        common = self._get_formio_common_parameters()
+        validate = self._get_formio_validate_parameters()
+        specific = {}
+        if self.question_type == "free_text":
+            specific = self._get_formio_textarea_parameters()
+        if self.question_type == "textbox":
+            specific = self._get_formio_textfield_parameters()
+        if self.question_type == "numerical_box":
+            specific = self._get_formio_number_parameters()
+        if self.question_type == "date":
+            specific = self._get_formio_datetime_parameters(is_datetime=False)
+        if self.question_type == "datetime":
+            specific = self._get_formio_datetime_parameters()
+        if self.question_type == "simple_choice":
+            specific = self._get_formio_radio_parameters()
+        if self.question_type == "multiple_choice":
+            specific = self._get_formio_selectboxes_parameters()
```

Comment:
I'd use `elif`'s here. Also a `else` with a warning that the question type is not supported?